gffquant/profilers/feature_quantifier.py

```
@@ -323,6 +352,17 @@ def finalise(
             )
 
         if self.aln_counter.get("aln_count"):
+            if self.adm is None:
+                self.adm = AnnotationDatabaseManager.from_db(self.db, in_memory=in_memory)
+
+            report_args = {
```

Ruff (0.8.0)

Local variable `report_args` is assigned to but never used

Remove assignment to unused variable `report_args`

(F841)